client/__tests__/CreateGroup/AddUser.test.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+import toJson from 'enzyme-to-json';
+import { AddUser } from '../../components/CreateGroup/AddUser.jsx';
+
+const props = {
+  fetchUsersRequest: jest.fn(),
+  addUserRequest: jest.fn(),
+  users: ['Ken', 'Ben', 'Ken'],
+  searchResult: {
+    pageCount: 6,
+    users: ['Samuel', 'Kachi', 'John']
+  }
+};
+describe('<AddUser />', () => {
+  it('Component should render correctly', () => {
+    const wrapper = shallow(<AddUser { ...props }/>);
+    const tree = toJson(wrapper);
+    expect(tree.type).toBe('div');
+    expect(tree.props.className).toBe('shift-right');
+  });
+  describe('Respond to change event', () => {
+    it('should update the state on select of group', () => {
+      const wrapper = shallow(<AddUser { ...props }/>);
+      const event = {
+        target: {
+          name: 'groupId',
+          value: '2',
+        }
+      };
+      wrapper.instance().onChange(event);
+      expect(wrapper.state().groupId).toEqual('2');
+    });
+  });
+  describe('Respond to change event', () => {
+    it('should update the state on select of a user', () => {
+      const wrapper = shallow(<AddUser { ...props }/>);
+      const event = {
+        target: {
+          name: 'usernames',
+          value: 'Ben',
+        }
+      };
+      wrapper.instance().onSelectUser(event);
+      expect(wrapper.state().usernames).toEqual(['Ben']);
+    });
+  });
+  describe('Test for onSubmit function', () => {
```

Comment:
### Nitpick
Your test descriptions should be more descriptive , for example instead of saying _Test for onSubmit function, should be called on button click_ you could go for something of this sort rather _Onsubmit function, should submit form values when submit button is clicked_